SensioLabs/Security/SecurityChecker.php

```diff
@@ -66,4 +75,56 @@ public function getCrawler()
     {
         return $this->crawler;
     }
+
+    /**
+     * Retrieves list of exclusions from external config file.
+     *
+     * @param string $configFile The path to the securityChecker.json file
+     *
+     * @return array An array of vulnerabilities to exclude
+     */
+    private function getSecurityExclusions($configFile)
+    {
+        try {
+            $config = json_decode(file_get_contents($configFile), true);
+
+            if ($config['exclusions'] && is_array($config['exclusions'])) {
+                return $config['exclusions'];
+            }
+        } catch (\Exception $e) {
+            throw new RuntimeException('Config file does not contain valid json.');
+        }
+
+        return [];
+    }
+
+    /**
+     * Removes specified exclusions from the checks.
+     *
+     * @param array $check      Security check results
+     * @param array $exclusions CVEs / links to remove
+     *
+     * @return array Filtered list of vulnerabilities
+     */
+    private function removeExclusions($check, $exclusions)
+    {
+        foreach ($check[1] as $entryKey => $entry) {
```

Comment:
Are there any docs that dictate the structure of `$checks`? Don't like using arbitrary array keys without some explanation around the expected structure.

Reply:
@jamie-infinity it's documented here https://github.com/graemechapman/security-checker/blob/dc270d5fec418cc6ac983671dba5d80ffaffb142/SensioLabs/Security/Crawler/CrawlerInterface.php#L24